### ISSUE TITLE
bug: create bucket on creation

### ISSUE THREAD

**Issue by philwinder** [combinator-ml/terraform-k8s-mlflow#12]
```
2021/07/19 08:07:14 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/usr/local/lib/python3.9/site-packages/mlflow/sklearn/__init__.py:790: UserWarning: Logging to MLflow failed: Failed to upload /tmp/tmpycohi0s_/model/MLmodel to mlflow-artifacts/0/af3f6cb59fdc4c5f815359495756e3bb/artifacts/model/MLmodel: An error occurred (NoSuchBucket) when calling the PutObject operation: The specified bucket does not exist"
```

I have some code somewhere for creating a bucket...

**Comment by lukemarsden** [comment]
this is how we used to do it, I think:

https://github.com/combinator-ml/terraform-k8s-mlflow/blob/76cb521be4a32ca021ed097f6e752c9792897bf0/conf/minio_values.yaml#L243

**Comment by philwinder** [comment]
Ah ok. That might only work with the old version of the chart you are using. The new operator one will be different.

Happy if it works for now.

Alternatively, here is the code I was thinking of: https://github.com/combinator-ml/terraform-k8s-stack-pachyderm-minio/blob/main/stack.tf#L20

**Comment by philwinder** [comment]
Fixed in e14e4d7a4cd5de66344d267a8001a3bd4f989d85
